src/js/grapl-cdk/lib/grapl-cdk-stack.ts

```diff
@@ -409,22 +409,25 @@ export class DGraphSwarmCluster extends cdk.NestedStack {
     private readonly dgraphSwarmCluster: Swarm;
 
     constructor(
-        parent: cdk.Construct, id: string, props: DGraphSwarmClusterProps
+        parent: cdk.Construct,
+        id: string,
+        props: DGraphSwarmClusterProps
     ) {
         super(parent, id);
 
         this.dgraphAlphaZone = new route53.PrivateHostedZone(
             this,
-            id + "-DGraphSwarmZone",
+            'DGraphSwarmZone',
             {
                 vpc: props.vpc,
-                zoneName: "alpha.dgraph.graplsecurity.com"
+                zoneName: 'alpha.dgraph.graplsecurity.com',
             }
         );
 
-        this.dgraphSwarmCluster = new Swarm(this, props.prefix + "-DGraphSwarmCluster", {
+        this.dgraphSwarmCluster = new Swarm(this, 'DGraphSwarmCluster', {
+            prefix: props.prefix,
             vpc: props.vpc,
-            internalServicePorts: [ec2.Port.tcp(5080), ec2.Port.tcp(7080)]
+            internalServicePorts: [ec2.Port.tcp(5080), ec2.Port.tcp(6080), ec2.Port.tcp(7080), ec2.Port.tcp(7081), ec2.Port.tcp(7082), ec2.Port.tcp(9080), ec2.Port.tcp(9081), ec2.Port.tcp(9082)],
```

Comment:
6080, 9080, 9081, and 9082 shouldn't be necessary to define here as they're external ports. We expose the external ports here: https://github.com/grapl-security/grapl/blob/staging/src/js/grapl-cdk/lib/grapl-cdk-stack.ts#L436

So *that* method should be updated to 

``` typescript
public allowConnectionsFrom(other: ec2.IConnectable): void {
    this.dgraphSwarmCluster.allowConnectionsFrom(other, ec2.Port.tcp(9080));
    this.dgraphSwarmCluster.allowConnectionsFrom(other, ec2.Port.tcp(9081));
    this.dgraphSwarmCluster.allowConnectionsFrom(other, ec2.Port.tcp(9082));
}```

Reply:
addressed in later commit